lib/salsify_rubocop/version.rb

```diff
@@ -1,3 +1,3 @@
 module SalsifyRubocop
-  VERSION = '0.52.1.1'.freeze
+  VERSION = '0.52.2'.freeze
```

Comment:
The versioning of this gem is a bit funky. It tracks the version of the Rubocop gem so this should be `0.52.1.2` (or we should upgrade rubocop).